config.yaml

```diff
@@ -112,3 +112,34 @@ languages:
           url: https://github.com/go-gitea/
           weight: 60
           pre: github
+
+  pt-br:
+    weight: 3
+    languageName: Português brasileiro
```

Comment:
Keep the language names for now. We can work on a better solution later on like flags or a select.